update acknowledgements/sponsorship (#18)
MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day: README.md

Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop materials.

> We would like to give a special thanks to our partner and primary sponsor for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).

Author: @armcdona would you like to change anything here?

Reviewer: This looks good to me.

Author: thanks!

> We would like to give a special thanks to our partner for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI), who is the primary sponsor of this workshop.

Author: @armcdona would you like to change anything here?

> We would like to give a special thanks to our partner for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI), who is the primary sponsor of this workshop.

Reviewer 2: ```suggestion
We would like to give a special thanks to our partner and primary sponsor for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
```

Reviewer 2: A suggestion for reducing the number of words that you are welcome to take or leave

<img
src="/logos/MolSSI_Logo.png"
title="Molecular Sciences Software Institute Logo" alt="Molecular Sciences Software Institute Logo"
style="float: left; height: 5em; " />

> This workshop has been made possible in part by a [grant](https://chanzuckerberg.com/eoss/proposals/mdanalysis-outreach-and-project-manager/) from the Chan Zuckerberg Initiative (CZI) DAF, a donor advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989) and through the support of the [Center for Biological Physics](https://cbp.asu.edu/) at Arizona State University.

Author: @jennaswa @fiona-naughton @yuxuanzhuang @IAlibay are we ok with mentioning CBP here and add the logo below?
